3rdParty/velocypack/src/Parser.cpp

```diff
@@ -118,6 +118,17 @@ int Parser::skipWhiteSpace(char const* err) {
   throw Exception(Exception::ParseError, err);
 }
 
+void Parser::increaseNesting() { 
```

Comment:
would we throw exception, not just leave nesting as it is if can't increase it more?